idem_azurerm/states/azurerm/redis/operations.py

```diff
@@ -284,6 +301,29 @@ async def present(
         )
 
     if "error" not in cache:
+        if wait_for_ready and action == "create":
+            if check_interval < 30 or check_interval > 600:
```

Comment:
why is anything under 30 seconds not long enough?

Reply:
There wasn't a particular reason other than tying to reduce the number of API calls someone may be making. I can change it to being anything over 0.

Reply:
I made the max 600 so people don't set it too high and wait much longer than they need to.